src/index.ts

```diff
@@ -174,8 +174,14 @@ async function initialBuild(): Promise<void> {
 
     const concurrencyLimit = pLimit(8);
     const globConfig = { nodir: true };
-    const svelteAndJsFiles = glob.sync('src/**/*.+(js|mjs|svelte)', globConfig);
-    const otherAssetFiles = glob.sync('src/**/*.!(js|mjs|svelte)', globConfig);
+    const svelteAndJsFiles = glob.sync(
+        'src/**/!(*+(spec|test)).+(js|mjs|svelte)',
+        globConfig
+    );
+    const otherAssetFiles = glob.sync(
+        'src/**/*.!(spec.js|test.js|js|mjs|svelte)',
```

Comment:
Good solution. I think we should actually avoid copying ts files too though, since i guess people could be writing tests in ts.

```suggestion
        'src/**/*.!(spec.[jt]s|test.[jt]s|[jt]s|mjs|svelte)',
```

^ I didn't test this yet... does it look right to you? 

Maybe if you could throw in a file like `snapshot/src/utils/something.ts` and `something.test.ts` for example, the snapshot test will still pass because they should not be copied to `dist`.

Reply:
I have added the following files:

- tests/snapshot/src/should-not-be-copied-to-dist.spec.js
- tests/snapshot/src/should-not-be-copied-to-dist.test.ts
- tests/snapshot/src/should-not-be-copied-to-dist.ts

and the suggested change works well.

You were right about new Svelte release too. They published a new release about 5 hours ago.  Would you like me to update the snapshot as well?

Reply:
Perfect! And yes, that would be great if you could update the snapshots to include the svelte update 👍